tests/test_visitors/test_tokenize/test_statements/test_inconsistent_comprehension.py

```diff
@@ -0,0 +1,277 @@
+import pytest
```

Comment:
What about generator expressions? They are not tested right now.

Reply:
Ah we had forgotten this syntax before now. We can add support for generators by including parenthesis as possible enclosing characters. The only potential issue would be if a for loop could potentially be contained in parenthesis, but this doesn't seem possible, as this could only happen within a function definition. While lambdas can be enclosed by parenthesis, they only allow for a single statement and implicitly exclude for loops. Does this reasoning sound correct?

Reply:
Yes, looks like this!